lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -6,7 +6,7 @@ class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmC
     expose :resize_confirm, :override_return => nil
     expose :resize_revert,  :override_return => nil
 
-    expose :validate_resize
+    expose :supports_resize?
```

Comment:
@gmcculloug thanks, I think we go with adding the wrapper method. :+1: for pointing out the alternative.

@gauravaradhye so, please add this wrapper and remove the `expose` call altogether